test-plugins/org.yakindu.sct.test.models/src/org/yakindu/sct/test/models/SCTUnitTestModels.java

```diff
@@ -1,125 +1,126 @@
-/**

```

Comment:
Something went wrong with the diff of SCTUnitTestModels maybe due to reformatting?